projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -122,9 +122,8 @@ export default function Edit( { attributes, setAttributes, clientId } ) {
 	const postId = useSelect( select => select( 'core/editor' ).getCurrentPostId() );
 
 	let loading = false;
-	const categories = useSelect( select =>
-		select( 'core/editor' ).getEditedPostAttribute( 'categories' )
-	);
+	const categories =
+		useSelect( select => select( 'core/editor' ).getEditedPostAttribute( 'categories' ) ) || [];
```

Comment:
Why does the tags `useSelect` pass the empty array (to only run once) and the categories doesn't? Essentially it looks to me like the categories should get reevaluated each render, while the tags only evaluates the first time...

Reply:
> Why does the tags useSelect pass the empty array (to only run once) 

You misread the code I think....